global.json

```diff
@@ -1,9 +0,0 @@
-{
-  "projects": [
-    "src",
-    "test/TestApps"
-  ],
-  "sdk": {
-    "version": "1.0.0-preview2-1-003180"
```

Comment:
Actually, I'm about to re-add global.json to our converted repos with sdk:version set to 1.0.0-preview4-004233. It's gonna take a week or so to move everyone up to RC3 and I don't want to break RC2 repos in the meantime.